catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-common/pom.xml

```diff
@@ -46,12 +46,16 @@
         </dependency>
         <dependency>
             <groupId>com.vividsolutions</groupId>
-            <artifactId>jts</artifactId>
+            <artifactId>jts-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts-io</artifactId>
         </dependency>
         <dependency>
             <groupId>org.jvnet.jaxb2_commons</groupId>
             <artifactId>jaxb2-basics-runtime</artifactId>
-            <version>1.11.1</version>
+            <version>0.11.0</version>
```

Comment:
❓Did we try using `0.11.1` or `0.12.0` instead? `0.11.0` is pretty old (2015-12-27).

Reply:
I did try `0.12.0`, it doesn't work unless you pull in the `2.6.1` versions of the org.jvnet artifacts, but that introduces its own set of issues because of how we're generating all of these schema artifacts. We need to remove all of these different JAXB jars and just have a single uber jar that builds them all.

They completely changed the API in version `0.12.0`. Most of the artifacts depend on `0.11.0`, so I used that version to get all the JAXB stuff to line up correctly. Once you start mixing and matching different versions than what the transitives are, you run into no end of compatibility issues.